addons/account/i18n/account.pot

```diff
@@ -14803,7 +14803,12 @@ msgstr ""
 
 #. module: account
 #: model:ir.model.fields,field_description:account.field_account_invoice_report__price_subtotal
-msgid "Untaxed Total"
+msgid "Untaxed Amount"
```

Comment:
pot not needed since you are doing a modification in master on a non localisation module :smile: